docs/implementation.adoc

```diff
@@ -357,6 +357,18 @@ public class AccurevSCMSource extends SCMSource {
                     new NonLocalizable("Streams")
                     // Better: Messages._AccurevSCMSource_StreamHeadCategory()
                 ),
+                new UncategorizedSCMSourceCategory(
+                    new NonLocalizable("Gated Streams")
+                    // Better: Messages._AccurevSCMSource_GatedStreamHeadCategory()
+                ),
+                new ChangeRequestSCMHeadCategory(
```

Comment:
These will need to have their own category classes... you can host them in the accurev plugin as they only make sense there.

If you want you should (i.e. I have not checked) be able to *extend* `ChangeRequestSCMHeadCategory` if they share the semantics of a change request